largestPrimeFactor.js

```diff
@@ -1,6 +1,46 @@
 exports.largestPrimeFactor = function(n) {
   let primeNumber = 0;
   // do your work here
+  let tempArr = [];
+
+  // prime number checker
+  function isPrime(num) {
+    if (num <= 1) {
+      // ignore 0 and 1
+      return false;
+    } else if (num <= 3) {
+      // 2 and 3 are prime numbers
+      return true;
+    } else if (num % 2 === 0 || num % 3 === 0) {
+      // 2 or 3 div
+      return false;
+    }
+
+    let i = 5;
+    while (i * i <= num) {
```

Comment:
It took me awhile to figure what's going on here. For complicated bits of code, you should have a comment that explains what going on below.